edb/server/compiler/errormech.py

```diff
@@ -31,6 +32,10 @@
 from edb.pgsql import types
 
 
+class SchemaNeededForError:
+    '''A sentinel used to signal that a particular error requires a schema.'''
+
+
 class PGError(enum.Enum):
```

Comment:
While you're here, could you please rename it to PGErrorCode?

Reply:
OK